Bottom: 58b5956bb7e9fa9dd5c90871d5ff8221b09c3767
Top:    3e45fca3abaf44a76e3e0ef8a3ec9e8e6d522144
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-06 02:33:35 +0100

Refresh of remove-hack-in-draw-c-which-br

---

diff --git a/src/draw.c b/src/draw.c
index 6dcfc9a..a7a1a74 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -694,7 +694,7 @@ DrawLayerGroup (int group, const BoxType *drawn_area)
           strcmp (Layer->Name, "route") == 0)
         rv = 0;
       if (layernum < max_copper_layer && Layer->On)
-        DrawLayerCommon (Layer, drawn_area, true);
+        dapi->draw_layer (Layer, drawn_area, NULL);
     }
   if (n_entries > 1)
     rv = 1;
